dss-api.yml

```diff
@@ -258,6 +258,79 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+    put:
+      summary: Create a bundle
+      description: |
+        Create a new bundle with a given UUID.  The list of files UUID+versions to be included must be provided.
+      parameters:
+        - name: uuid
+          in: path
+          description: A RFC4122-compliant ID for the file.
+          required: true
+          type: string
+          pattern: "[A-Za-z0-9]{8}-[A-Za-z0-9]{4}-[A-Za-z0-9]{4}-[A-Za-z0-9]{4}-[A-Za-z0-9]{12}"
+        - name: version
+          in: query
+          description: Timestamp of bundle creation in RFC3339.
+          required: false
+          type: string
+          format: date-time
+        - name: replica
+          in: query
+          description: Replica to write to.
+          required: true
+          type: string
+          pattern: "^(AWS|GC|Azure)$"
+        - name: extras
+          in: body
+          required: true
+          schema:
+            type: object
+            properties:
+              files:
+                type: array
+                items:
+                  type: object
+                  properties:
+                    uuid:
+                      description: A RFC4122-compliant ID for the file.
+                      type: string
+                      pattern: "[A-Za-z0-9]{8}-[A-Za-z0-9]{4}-[A-Za-z0-9]{4}-[A-Za-z0-9]{4}-[A-Za-z0-9]{12}"
+                    version:
+                      description: Timestamp of file creation in RFC3339.
+                      type: string
+                      format: date-time
+                    name:
+                      description: Name of the file.
+                      type: string
+                      pattern: "^[^/\\\\]+$"
+                    indexed:
+                      description: True iff this file should be indexed.
+                      type: boolean
```

Comment:
We may want to sniff/infer indexed status in the future (for example from content-type) but for now I'm fine either way.